tests/unit/mixins/tooltip-for-dropdown-test.js

```diff
@@ -0,0 +1,12 @@
+import EmberObject from '@ember/object';
+import TooltipForDropdownMixin from 'code-corps-ember/mixins/tooltip-for-dropdown';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | tooltip for dropdown');
+
+// Replace this with your real tests.
+test('it works', function(assert) {
+  let TooltipForDropdownObject = EmberObject.extend(TooltipForDropdownMixin);
+  let subject = TooltipForDropdownObject.create();
+  assert.ok(subject);
+});
```

Comment:
If we're not writing tests as part of this PR, it could be an option to at least add a blank `skip`

https://api.qunitjs.com/QUnit/skip

Along the lines of

```Javascript
skip('description of behavior A', function(assert) {});
skip('description of behavior B', function(assert) {});
```

This will report those tests as skipped in the output and keep us aware.

Also, while it does not apply here, take note of `QUnit.todo` - https://api.qunitjs.com/QUnit/todo